File: 2025/학부연구생/Code/Grading problems/class_v2.2/C_answer_Code/problem5.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    FILE *input = fopen("input.txt", "r");
    FILE *output = fopen("output.txt", "w");

    if (input == NULL || output == NULL) {
        printf("파일 열기 오류!\n");
        return 1;
    }

    int a, b;
    fscanf(input, "%d %d", &a, &b);

    int result = abs(a - b); // 절댓값 계산
    fprintf(output, "%d\n", result);

    fclose(input);
    fclose(output);
    return 0;
}

File: 2025/학부연구생/Code/Python programming_grade/class_v2.2/2025_python_Answer_re/problem5.py
# input.txt 파일에서 입력 읽기
with open('input.txt', 'r') as f:
    n = int(f.read().strip())

# 짝수인지 홀수인지 판별하기
if n % 2 == 0:
    result = "Even"
else:
    result = "Odd"

# output.txt 파일에 결과 쓰기
with open('output.txt', 'w') as f:
    f.write(result)